q

```diff
@@ -0,0 +1,35 @@
+[33mcommit 28e145061b755efe04b9de2813bc7ee9adea232f[m[33m ([m[1;36mHEAD -> [m[1;32mGrid_Opdracht_RemcoS[m[33m)[m
```

Comment:
Hoi! Wat is dit losse 'q' bestand in jouw repo? Daar lijkt een vreemd bijverschijnsel te zijn gebeurd? (ik heb dit zelf i.e.g. niet).

Reply:
Is gebeurd tijdens het willen pushen naar gitHub, weet niet exact hoe en wist even niet hoe dit verwijderd moest worden.